core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -430,6 +430,64 @@ public void dayOfMonth() {
     assertEquals(integerValue(8), eval(expression));
   }
 
+  public void testDayOfMonthWithUnderscores(FunctionExpression dateExpression, int dayOfMonth) {
```

Comment:
`private`

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/194/commits/878a6a152955d3cbbc1a21c85ff2d900a6bc785c